src/components/BusesTable.js

```diff
@@ -53,7 +55,11 @@ const BusesTable = () => {
     const fetchData = async () => {
       try {
         setloading(true);
-        const response = await axios.get('/buses');
+        const response = await axios.get('/buses', {
+          params: {
+            relation: true
+          }
+        });
         setbuses(response.data.data);
       } catch (error) {
         toast(error.message, { type: 'error' });
```

Comment:
We can also check for server response error for the message property. Check the server docs on errors schema